en/components/general/update-guide.md

```diff
@@ -51,6 +51,35 @@ Unfortunately not all changes can be automatically updated. Changes below are sp
 
 For example: if you are updating from version 6.2.4 to 7.1.0 you'd start from the "From 6.x .." section apply those changes and work your way up:
 
+## From 14.1.x to 14.2.x
+### General
+- **Breaking Change** - All **igx** prefixes are replaced with **ig** prefixes. For example, **igx-**typography is now **ig-**typography.
+
+- Add this line in your `angular.json` file:
+```
+"stylePreprocessorOptions": {
+    "includePaths": ["node_modules"]
+}
+```
+to access the `igniteui-theming package`, without writing the full path. You should have the igniteui-theming package installed in your `node_modules` folder first. If it's not installed automatically, use the `npm install igniteui-theming` command in your project directory.
+
+### Themes
+- **Breaking Change** - All `$grays` variable names are switched with `$gray`. <br />
+If you're using the **color function** to get a gray shade, use it like this: `color($green-palette, 'gray', 800);`
+
+- **Breaking Change** - **Generating css variables** for the palette is now done by the **palette mixin**, not by the **palette-vars mixin**.
+
+- **Breaking Change** - The **palette function** now **requires the surface color** and the **grays parameter** is renamed to **gray**. The gray color can be generated automatically based on the surface color, or you can set one yourself. When you're generating a palette, you have to keep in mind that there are no longer default values for **info, success, error and warn** colors. You have to set the explicitly if you want to use them.
```

Comment:
```suggestion
- **Breaking Change** - The **palette function** now **requires the surface color** and the **grays parameter** is renamed to **gray**. The gray color will be generated automatically based on the surface color, or you can set it yourself. When you're generating a palette, you have to keep in mind that there are no longer default values for **info, success, error and warn** colors. They have to be set explicitly if you want to use them in custom created palettes.
```

This should also showcase how users can pick the info, success, error, and warn colors from an existing palette when creating a custom palette if they don't want to come up with the colors themselves.